gm4_guidebook/generate_guidebooks.py

```diff
@@ -2029,11 +1889,19 @@ def generate_toast_model(book: Book, ctx: Context) -> Model:
   ctx.assets[f"{ctx.project_id}:gui/guidebook/{book.id}"] = Texture(img)
 
   # create model for new texture
+  textures = {
+    "layer0": f"{ctx.project_id}:gui/guidebook/{book.id}"
+  }
+   # some items will tint layer0, so we special case those here.
+  if book.icon.id.removeprefix("minecraft:") in ("leather_helmet", "leather_chestplate", "leather_leggings", "leather_boots", "leather_horse_armor", "potion", "splash_potion", "lingering_potion", "tipped_arrow"):
+    textures = {
+      "layer0": "gm4_guidebook:gui/font_magic/blank",
+      "layer1": f"{ctx.project_id}:gui/guidebook/{book.id}"
```

Comment:
Both mecha and minecraft complain because the `gui/font_magic/blank` texture doesn't exist. Instead it is currently at `font_magic/blank`. However even after changing this, it still has the problem where these textures are not loaded to the `blocks.json` atlas, which I believe is required to render the textures on item models.

Reply:
I've fixed the issue in-game, but can't figure out why mecha still complains. 
(This is pasted from commit e7ccfa2)
> Mecha still complains that the texture is missing, but the built resource pack works in game without throwing a warning. I've even tried modifying the `minecraft/atlases/block.json` in the `base` module, but it still shows a warning

Reply:
These warnings are from other modules, which won't have the guidebook textures loaded when they build. I would recommend using `gm4:item/empty` from the base module, or if the image size is an issue, moving `font_magic/blank` to `gm4:gui/advancement/blank` in the base.

Reply:
Oh that's perfect. I didn't know we already had an empty texture laying around.